RitoxicGame/space-trader
# making the build work

## What did you add in this branch?

Minor script and UI changes to fix the playtest build
## How can we test these changes?

Was tested during playtest:
see also the current playtest build on google drive.
## Relevant screenshots (if applicable):


## What issue is this PR associated with?

N/A
## What week is this due by?

Week 3: Playtest 2/Final Build
